css/style.css

```diff
@@ -0,0 +1,632 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap");
+@import url("../assets/fonts/stylesheet.css");
+:root {
+  --color-white: #fff;
+  --color-background-dark: #f5f5f5;
+  --color-primary: #ec5242;
+  --color-dark: #272a31;
+  --color-dark-light: #3b3e45;
+  --color-light: #d3d3d3;
+  --color-text: #191919;
+  --color-header-text: #555;
+  --color-bck: #414246; }
+
+* {
+  margin: 0;
+  padding: 0; }
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box; }
+
+html {
+  font-size: 62.5%; }
+
+body {
+  background-color: var(--color-background-dark);
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 1.6rem;
+  line-height: 2.4rem; }
+  @media (max-width: 767px) {
```

Comment:
- [ ]  Your  CSS styling should be  built as **mobile-first**,  please don't use max-width in your design, and you should not use media queries in your mobile version, you should remove all media queries in your mobile version 👍🏿

Reply:
@prolajumokeoni Please could you explain what you mean? I have just one CSS file for both mobile and desktop

Reply:
@prolajumokeoni  After some thinking, I understand now. I have implemented the above changes